packages/documentation-framework/scripts/webpack/webpack.base.config.js

```diff
@@ -55,20 +55,12 @@ module.exports = (_env, argv) => {
             options: {
               cacheDirectory: '.cache/babel',
               cacheCompression: false,
-              presets: [['@babel/preset-env', {
-                loose: true,
-                corejs: 3,
-                useBuiltIns: 'entry',
-                exclude: ['transform-regenerator', 'transform-async-to-generator'],
```

Comment:
I've removed these transforms, as they are now implicitly no longer included by updating the `targets` field, as well as updating the [browserlist database](https://github.com/browserslist/update-db?tab=readme-ov-file#update-browserslist-db) (change included in updated lockfile).